NMEA2000.cpp

```diff
@@ -534,7 +537,13 @@ void tNMEA2000::HandleISORequest(const tN2kMsg &N2kMsg) {
       case 126998L: /* Configuration information */
         SendConfigurationInformation(iDev);
         break;
-       // ToDo: Write default handler so that user can add their own.
+      default:
+         if (RqstHandler!=0 ) RqstHandler(N2kMsg);     /* See if user wants to handle it */
```

Comment:
That is an idea, would save us from replicating the NAK code in our handler.
Perhaps a downside is the RqstHandler and the _MsgHandler functions would then be asymmetrical (One BOOL and the other VOID )